pkg/provider/resource_cluster.go

```diff
@@ -1226,3 +1221,26 @@ func StringSliceToSet(items *[]string) types.Set {
 
 	return types.SetValueMust(types.StringType, eles)
 }
+
```

Comment:
add a one-liner comment to describe purpose of the function.

Reply:
I moved this function with a comment into common.go in a later PR https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/533